addons/auth_signup/tests/test_reset.py

```diff
@@ -0,0 +1,19 @@
+# rerun TestWebLogin tests with auth_signup installed
+from odoo.tests.common import HttpCase
+
+class TestResetPassword(HttpCase):
+    def test_password(self):
```

Comment:
Would add a small docstring explaining what (and why) we are doing this test, as it may not be obvious in a few months (having a docstring explaining the test's purpose allows to avoid having to blame and find the message)